venia-integration-tests/src/tests/pageBuilder/integrationTests/buttons.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('verify pagebuilder buttons content is rendered correctly', () => {
+    it('verify buttons content', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', {
+            fixture: 'pageBuilder/buttons/buttons.json'
+        }).as('getCMSMockData');
+        cy.visit('/');
```

Comment:
Use `visitHomePage` from  https://github.com/magento-obsessive-owls/pwa-studio/blob/PWA-1542/venia-integration-tests/src/support/routesCommands.js

Reply:
```suggestion
        cy.visitHomePage()
```

Reply:
Done, thank you